src/MyTested.WebApi/Builders/HttpConfigurationBuilder.cs

```diff
@@ -86,12 +88,41 @@ public IHttpConfigurationBuilder WithBaseAddress(string baseAddress)
             return this;
         }
 
+        /// <summary>
+        /// Sets custom inline constraint resolver to http configuration
+        /// </summary>
+        /// <param name="inlineConstraintResolver">Custom route constraint resolver to use.</param>
+        /// <returns>New HTTP configuration builder.</returns>
+        public IHttpConfigurationBuilder WithInlineConstraintResolver(IInlineConstraintResolver inlineConstraintResolver)
+        {
+            var config = new HttpConfiguration();
```

Comment:
This code is the same as `MyWebApi.IsUsingDefaultHttpConfiguration`. Extract a method which is reused in both places.